ql/src/test/results/clientpositive/llap/constprog_dpp.q.out

```diff
@@ -84,12 +84,13 @@ Stage-0
                         Select Operator [SEL_40] (rows=1 width=4)
                           Output:["_col0"]
                           TableScan [TS_24] (rows=1 width=4)
-                            Output:["id"]
+                            default@tb2,tb2,Tbl:COMPLETE,Col:NONE,Output:["id"]
                   <-Map 6 [CONTAINS] vectorized, llap
                     Reduce Output Operator [RS_45]
                       Limit [LIM_44] (rows=1 width=2)
                         Number of rows:1
                         Select Operator [SEL_43] (rows=1 width=0)
                           Output:["_col0"]
                           TableScan [TS_29] (rows=1 width=0)
+                            default@tb2,tb2,Tbl:PARTIAL,Col:COMPLETE
```

Comment:
This seems different than the stats above. Do you know if it is expected? May be worth checking in a follow-up.

Reply:
I confirmed that this is expected. I compared this plan against master (with explain.user set to false) and there is no difference in the plan.